requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest>=2.6.4
+pycrypto>=2.6.1
```

Comment:
Why do we need a `requirements.txt` in the package like this? All installation requirements are listed in `setup.py` (in `install_requires`), so when library users do `pip install pyope` those requirements will be installed as well.